agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/view/mobile/AgendaEventDetailsMobileToolbar.vue

```diff
@@ -1,25 +1,28 @@
 <template>
-  <div class="d-flex flex-row py-2">
-    <v-avatar
-      height="32"
-      min-height="32"
-      width="32"
-      min-width="32"
-      max-width="32"
-      size="32"
-      class="mx-3 my-auto spaceAvatar space-avatar-header">
-      <v-img :src="ownerAvatarUrl" />
-    </v-avatar>
-    <div class="d-flex flex-grow-1 flex-column align-left">
```

Comment:
Instead of all layout changing, you need just to change this line from : 
`    <div class="d-flex flex-grow-1 flex-column align-left">
`
into : 
`
    <div class="d-block text-truncate flex-grow-1 align-left">
`